src/SFA.DAS.ApplyService.InternalApi.Types/GatewayOrganisationTypes.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SFA.DAS.ApplyService.InternalApi.Types
+{
+    public class GatewayOrganisationTypes
```

Comment:
Could you rename the class to RoatpOrganisationTypes?  I've created the same class on a branch off master so we need to make sure we don't get two classes with the same constants.